### PR TITLE
Generate typescript files

### PR DESCRIPTION
**Pull request by dpgil**

### CODE REVIEW

**Comment by dpgil** [comment]
@jbadlato

File: scripts/generate.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const formatsWithTeams = ['gen7anythinggoes',
 // formats without teams, plus the above
 const formats = ['gen7randombattle'].concat(formatsWithTeams);
 
-const languages = ['nodejs'];
+const languages = ['typescript', 'nodejs'];
```
Comment (jbadlato) [inline scripts/generate.js:20]: Love that it's an option, nice

**Comment by jbadlato** [comment]
Please also update the list of supported languages in the README

**Comment by dpgil** [comment]
i got u

File: scripts/generate.js (hunk 1 above)
Comment (dpgil) [reply scripts/generate.js:20]: thranks

**Pull request merged**